R/03-modelResults2DKernel.R

```diff
@@ -63,11 +63,15 @@ modelResults2DKernelUI <- function(id, title = "", asFruitsTab = FALSE){
           selectInput(inputId = ns("Weighting"),
                       label = "Weighting variable (optional):",
                       choices = c("")),
-          checkboxInput(inputId = ns("kMeans"),
-                        label = "Do k-means clustering",
-                        value = FALSE, width = "100%"),
```

Comment:
Before there was the option to run the model without clustering:

![image](https://user-images.githubusercontent.com/16759098/228483266-a6f6ebb2-f6e3-4f85-b84f-8b04f804b89f.png)

Are now clusters always calculated? Would it make sense to add an option "none" and use this per default? (If so, the slider "No. of clusters" should be hidden.)

![image](https://user-images.githubusercontent.com/16759098/228483714-95c33503-b6bb-4b68-83d5-8758bb0ba831.png)

Reply:
You can still run it without clustering if you remove the selection in Cluster Method (using backspace). This is the default when you start the app. Do you think another option "none" is more intuitive. The way it is currently implemented is the same as for the previous inputs (e.g. Weighting variable (optional):)

Reply:
Ok, then keep it as it is to be consistent with the other input fields. :+1: